<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scheduler</class>
 <widget class="QWidget" name="Scheduler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1651</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>调度分析器</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>511</width>
     <height>391</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="scheduler_args_tab">
    <attribute name="title">
     <string>调度配置</string>
    </attribute>
    <widget class="QTableWidget" name="scheduler_args_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="schedule_result_tab">
    <attribute name="title">
     <string>调度结果</string>
    </attribute>
    <widget class="QTableWidget" name="schedule_result_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="DAG_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>481</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>DAG</string>
   </property>
   <widget class="QLabel" name="DAG_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>451</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="system_group">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>1111</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>系统模型图</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QScrollArea" name="system_scrollarea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1089</width>
         <height>267</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gantt_group">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>1111</width>
     <height>621</height>
    </rect>
   </property>
   <property name="title">
    <string>甘特图</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QScrollArea" name="gantt_scrollarea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1089</width>
         <height>587</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="start_btn">
      <property name="text">
       <string>▶️</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="pause_btn">
      <property name="text">
       <string>⏸️</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="backward_btn">
      <property name="text">
       <string>↩️</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="forward_btn">
      <property name="text">
       <string>↪️</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="frequency_btn">
      <property name="text">
       <string>⏱️</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="simtime_label">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SimTime:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="simtime"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
